src/Microsoft.AspNet.Diagnostics/Views/ErrorPage.cshtml

```diff
@@ -94,18 +94,15 @@
                                     <h3>@frame.Function in <code title="@frame.File">@System.IO.Path.GetFileName(frame.File)</code></h3>
                                 }
 
-                                @if (frame.Line != 0 && frame.ContextCode !=null && frame.ContextCode.Any())
+                                @if (frame.Line != 0 && frame.ContextCode.Any())
```

Comment:
Q: Why in `CompilationErrorPage.cshtml` do we check `ContextCode != null` and now we don't here?